### PR TITLE
[18.06] treewide: replace old wiki links

### LINKED ISSUE #10740: ddns-scripts: old link to wiki.openwrt.org
The ddns-scripts still contain a link to the old wiki:
https://github.com/openwrt/packages/blob/cce63a697bbc086f19d7dcb14111bd17d66a5902/net/ddns-scripts/Makefile#L39

Old URL: http://wiki.openwrt.org/doc/howto/ddns.client
New URL: https://openwrt.org/docs/guide-user/services/ddns/client

Please update to the new URL.

### PR DESCRIPTION
**Pull request by leonghui**
Backport of #10762, fixes #10740 

Maintainers: @champtar @yousong @dibdot 
Compile tested: N/A
Run tested: N/A

Description:
Replaced all references to "wiki.openwrt.org" with links to the latest wiki pages.

Signed-off-by: Leong Hui Wong <wong.leonghui@gmail.com>

### CODE REVIEW

File: admin/zabbix/files/mac80211 [hunk 1]
```diff
@@ -1,7 +1,7 @@
-#see http://wiki.openwrt.org/doc/howto/zabbix for ready to use templates
+#see https://openwrt.org/docs/guide-user/services/network_monitoring/zabbix for ready to use templates
 
 # If you want to know the exact meaning of an UserParameter, you can search in the ieee80211 standard:
-# http://standards.ieee.org/getieee802/download/802.11-2012.pdf
+# https://standards.ieee.org/getieee802/download/802.11-2012.pdf
```
Comment (BKPepe) [inline admin/zabbix/files/mac80211]: 404 page not found.

File: README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@
 
 This is the OpenWrt "packages"-feed containing community-maintained build scripts, options and patches for applications, modules and libraries used within OpenWrt.
 
-Installation of pre-built packages is handled directly by the **opkg** utility within your running OpenWrt system or by using the [OpenWrt SDK](http://wiki.openwrt.org/doc/howto/obtain.firmware.sdk) on a build system.
+Installation of pre-built packages is handled directly by the **opkg** utility within your running OpenWrt system or by using the [OpenWrt SDK](https://openwrt.org/docs/guide-developer/obtain.firmware.sdk) on a build system.
```
Comment (BKPepe) [inline README.md]: The site which you mentioned says that's an old one and that you should use
https://openwrt.org/docs/guide-developer/using_the_sdk

```suggestion
Installation of pre-built packages is handled directly by the **opkg** utility within your running OpenWrt system or by using the [OpenWrt SDK](https://openwrt.org/docs/guide-developer/using_the_sdk) on a build system.
```

File: admin/zabbix/files/mac80211 (hunk 1 above)
Comment (leonghui) [reply admin/zabbix/files/mac80211]: Oops thanks catching this. I guess the new link should be https://standards.ieee.org/standard/802_11-2016.html (supersedes IEEE 802.11-2012) but I can't get past the paywall. Will need to update in master and 19.07 too.

File: README.md (hunk 2 above)
Comment (leonghui) [reply README.md]: Yes I saw this earlier but I figured the link is also outdated in 19.07 and master. Will fix this for 18.06 for now.

**Pull request merged**